xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -342,7 +342,21 @@
   #foreach($item in $items)
     #gridresult_buildRowJSON($item $rows)
   #end
-  #set($discard = $map.put('rows', $rows))
+  ## When the total number of rows is not bigger than the limit, we remove the obfuscated results from the 
```

Comment:
Feels like this part (and maybe more I did not chech) should be moved into a macro in some template and used both here and in getdocument.vm.